libparsec/crates/client/src/workspace/store/resolve_path.rs

```diff
@@ -646,3 +664,108 @@ pub(crate) async fn resolve_path_for_reparenting(
         }
     }
 }
+
+/// Retrieve the path and the confinement point of a given entry ID.
+/// Both information are acquired without locking, so keep in mind that
+/// they could get outdated by the time you use them. This is fine however,
+/// since this is typically used for providing information to the user
+/// (e.g. for file and folder stats)
+pub(crate) async fn retrieve_path_from_id(
+    store: &super::WorkspaceStore,
+    entry_id: VlobID,
+) -> Result<(FsPath, PathConfinementPoint), ResolvePathError> {
+    // Initialize the results
+    let mut parts = Vec::new();
+    let mut confinement: Option<VlobID> = None;
```

Comment:
```suggestion
    let mut confinement = PathConfinementPoint::None;
```

Directly use the specific type

Reply:
done